ground/src/main/java/com/google/android/ground/util/StreamUtil.kt

```diff
@@ -22,26 +22,11 @@ import timber.log.Timber
 
 /** Methods for working with and manipulating [java8.util.stream.Stream]. */
 object StreamUtil {
-  /**
-   * Executes the specified Supplier, writing throw Error exceptions to debug logs. If an error
-   * occurs, an empty Stream is returned, otherwise a Stream with only the result of the Supplier is
-   * returned.
-   */
-  @JvmStatic
-  fun <R> logErrorsAndSkip(supplier: Supplier<R>): Stream<R> {
-    return try {
-      StreamSupport.stream(setOf(supplier.get()))
-    } catch (e: RuntimeException) {
-      Timber.e(e)
-      StreamSupport.stream(emptySet())
-    }
-  }
-
   fun <R> logErrorsAndSkipKt(supplier: Supplier<R>): Iterable<R> {
```

Comment:
sorry for the out-of-diff comment, but do we need the `Kt` suffix here?

Reply:
Fixed!